flink-connector-http/src/main/java/org/apache/flink/connectors/http/HttpSinkBuilder.java

```diff
@@ -1,20 +1,39 @@
-package com.getindata.connectors.http;
+/*
+ *
+ *  * Licensed to the Apache Software Foundation (ASF) under one
+ *  * or more contributor license agreements.  See the NOTICE file
+ *  * distributed with this work for additional information
+ *  * regarding copyright ownership.  The ASF licenses this file
+ *  * to you under the Apache License, Version 2.0 (the
+ *  * "License"); you may not use this file except in compliance
+ *  * with the License.  You may obtain a copy of the License at
+ *  *
+ *  *     http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS,
+ *  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  * See the License for the specific language governing permissions and
+ *  * limitations under the License.
+ *
+ */
 
-import java.util.Optional;
-import java.util.Properties;
+package org.apache.flink.connectors.http;
```

Comment:
I believe it should be `package org.apache.flink.connector.http`. (`connectors` -> `connector`)